backend/fms_core/models/container.py

```diff
@@ -87,14 +87,9 @@ def add_error(field: str, error: str):
                 if not errors.get("coordinates") and not errors.get("location") \
                         and not parent_spec.coordinate_overlap_allowed:
                     # Check for coordinate overlap with existing child containers of the parent
-                    try:
-                        check_coordinate_overlap(self.location.children, self, self.location)
-                    except CoordinateError as e:
-                        add_error("coordinates", str(e))
-                    except Container.DoesNotExist:
```

Comment:
Despite me disapproving with that behaviour some operations are expected to be performed (and are faster) using exception catching in python. You may have to accept that some things are done that way. There is as far as i know always a clean way around but it might force some extra operations.

Reply:
I've reverted the commit